multinet/db.py

```diff
@@ -47,7 +53,20 @@
 def db(name: str) -> StandardDatabase:
     """Return a handle for Arango database `name`."""
     return arango.db(
-        name, username="root", password=os.environ.get("ARANGO_PASSWORD", "letmein")
+        name,
+        username="root",
+        password=os.environ.get("ARANGO_PASSWORD", "letmein"),
+        verify=True,
+    )
+
+
+def read_only_db(name: str) -> StandardDatabase:
+    """Return a read-only handle for the Arango database `name`."""
+    return arango.db(
+        name,
+        username=str(os.environ.get("ARANGO_READONLY_USERNAME", "")),
+        password=str(os.environ.get("ARANGO_READONLY_PASSWORD", "")),
```

Comment:
This doesn't seem right. The fallback case essentially guarantees runtime failure.

Can we perhaps think about setting the username statically (since we do this for the root user already)--something like `readonly` or `reader`? And it seems to me the default password should also be `letmein` so that simple, local setups are easier to experiment with.

Reply:
The guaranteed runtime failure was by design, as if we're missing credentials we'll want to know, and I didn't want to create a multinet error for it, as it's not useful for the user to know this, just us. However, it would also necessitate that we've provisioned the db with this user, so it would also fail in that case with defaults. I think your idea is good, I can add it.